### DISCUSSION TITLE
Check remote cache for sucessful pipelines?

### DISCUSSION THREAD

**Discussion by Scalahansolo** [vercel/turborepo#5091] [Help] [answered]
I have this idea for my CI pipelines to be able to avoid ever having to run `yarn` during CI for builds / testing.

My theoretical flow is...

- Build new feature in a particular scope locally
- Run build / test locally (both succeed) and update the remote cache
- Open a PR
- PR first checks which targets have changed using something along the lines of the `--dry-run` feature. I.e. https://github.com/marketplace/actions/turbo-changed
- It find that one of my packages has changed
- I than call some API endpoint in the remote cache with my gitsha and package I changed
- It responds that the build and test pipelines have cache hits with successful outputs. 
- CI finishes

In the event there are no cache hits with successful results, it goes through the typical flow and `yarn installing` followed by the appropriate Turbo commands.

**Comment by anthonyshew** [comment, marked as answer]
Sounds like you can leverage `turbo-ignore` here: https://turbo.build/repo/docs/core-concepts/monorepos/skipping-tasks
